pkg/reconcilermanager/controllers/rootsync_controller.go

```diff
@@ -956,12 +956,17 @@ func (r *RootSyncReconciler) manageRBACBindings(ctx context.Context, reconcilerR
 	return nil
 }
 
-func (r *RootSyncReconciler) upsertSharedClusterRoleBinding(ctx context.Context, name, clusterRole string, reconcilerRef types.NamespacedName) error {
+func (r *RootSyncReconciler) upsertSharedClusterRoleBinding(ctx context.Context, name, clusterRole string, reconcilerRef, rsRef types.NamespacedName) error {
 	crbRef := client.ObjectKey{Name: name}
 	childCRB := &rbacv1.ClusterRoleBinding{}
 	childCRB.Name = crbRef.Name
 
+	labelMap := ManagedObjectLabelMap(r.syncKind, rsRef)
```

Comment:
In the 1-pager, sync-namespace can be omitted for RootSync resources. Is it still the case?

Reply:
In my first pass I did that, but discovered that this seems to break many of the metrics assertions which apparently rely on the sync-namespace label. I'm planning to update the 1-pager accordingly. I'm leaving the sync-namespace out of this change as it appears it may be a breaking change.

Reply:
ACK